tests/test_subscriptions.py

```diff
@@ -14,6 +14,8 @@
 import connexion.apis.abstract
 import requests
 
+from tests.auth_tests import TestAuthMixin
```

Comment:
Is `TestAuthMixin` used anywhere else?

Reply:
currently the tests are broken, I'm working on fixing them.

Reply:
it is now used many places